pkg/broker/broker.go

```diff
@@ -718,3 +724,35 @@ func (a AnsibleBroker) AddSpec(spec apb.Spec) (*CatalogResponse, error) {
 	service := SpecToService(&spec)
 	return &CatalogResponse{Services: []Service{service}}, nil
 }
+
+func (a AnsibleBroker) RemoveSpec(specID string) error {
+	spec, err := a.dao.GetSpec(specID)
+	if client.IsKeyNotFound(err) {
+		return ErrorNotFound
+	}
```

Comment:
I was thinking that alerting the user to a spec not being found from the tool, could give the users valuable feedback. But I do see where this could be considered a noop as well. Does anyone have super strong opinions?